colossalai/kernel/cuda_native/csrc/gptq/cu_compat.cuh

```diff
@@ -0,0 +1,58 @@
+// Adapted from turboderp exllama: https://github.com/turboderp/exllama
```

Comment:
I notice you use code from exllama. Do you need to add their license to ours?

Reply:
I added their license in our LICENSE.